src/builder_ext.rs

```diff
@@ -314,11 +315,17 @@ impl UpdateCollectionClusterSetupRequestBuilder {
 }
 
 impl MoveShardBuilder {
-    pub fn new(shard_id: u32, from_peer_id: u64, to_peer_id: u64) -> Self {
+    pub fn new(
+        shard_id: u32,
+        from_peer_id: u64,
+        to_peer_id: u64,
+        method: ShardTransferMethod,
```

Comment:
The method should be optional, so we probably want to revert this change.

Instead, we want `UpdateCollectionClusterSetupRequestBuilder.operation` to be required.

Reply:
Oh my bad, misunderstood your message. I'll revert it and implement it correctly!

Reply:
No problem. Thanks!